fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/AbstractMaterializedViewAggregateRule.java

```diff
@@ -267,7 +273,7 @@ private Function rollup(AggregateFunction queryFunction,
                 if (!(mvExprShuttled instanceof Function)) {
                     continue;
                 }
-                if (isAggregateFunctionEquivalent(queryFunction, (Function) mvExprShuttled)) {
+                if (isAggregateFunctionEquivalent(queryFunction, queryFunctionShuttled, (Function) mvExprShuttled)) {
```

Comment:
add more comment to explain each parameters of function `rollup`, `isAggregateFunctionEquivalent` and `extractViewArguments`

Reply:
OK,I will add more comment.